source/LogbookPanel.h

```diff
@@ -60,9 +61,13 @@ class LogbookPanel : public Panel {
 	// Other months available for display:
 	std::vector<std::string> contents;
 	std::vector<Date> dates;
+
+	Point hoverPoint;
 	
 	// Current scroll:
 	double scroll = 0.;
+	double categoryScroll = 0.; 
+	mutable double maxCategoryScroll = 0.;
 	mutable double maxScroll = 0.;
```

Comment:
Just organizing these variables to match the organization of ShopPanel. (Scroll variables in some order then max scroll variables in that same order.)
```suggestion
	double categoryScroll = 0.; 
	double scroll = 0.;
	mutable double maxCategoryScroll = 0.;
	mutable double maxScroll = 0.;
```